Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.8.6-buster as base
+RUN pip install "poetry==1.1.4"
+WORKDIR /todo-app
+EXPOSE 5000
+
+FROM base as production
+COPY . /todo-app

```

Comment:
To improve build layer caching, you can split this into copying the poetry files here, and then copying `.` after the poetry install.

Reply:
Done!